ipv8/test/base.py

```diff
@@ -144,6 +147,8 @@ def introduce_nodes(self):
         yield self.deliver_messages()
 
     def temporary_directory(self):
-        d = os.path.join("temp", self.__class__.__name__ + str(int(time.time())))
+        rndstr = ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(10))
```

Comment:
Could you use `uuid` for this like in  https://github.com/Tribler/tribler/pull/3869/files#diff-c2974363c4135bb26324c12fb3724032R119 ? Also, please prefix the folder name with `test_` or `temp_` or something (or put all of the dirs inside the `temp` dir like previously).

Reply:
if we put everything inside a single `temp` dir, we return to the same problem - the problem of one common resource shared by each test.

Reply:
and we don't want to use UUIDs because they are too long. On Windows there is a limitation of 255 characters for the whole filesystem *path*.

Reply:
@xoriole ⏫ 

Alright, could you prefix the folder with something that shows it's a temporary directory though? This makes it easier to delete it, should something go wrong with its deletion.

Reply:
fine, just forgot to add the prefix